requirements/edx/base.txt

```diff
@@ -71,6 +71,7 @@ path.py==7.2
 piexif==1.0.2
 Pillow==3.1.1
 polib==1.0.3
+py2neo==3.1.1
```

Comment:
Yeah, not sure when we should be separating requirements out or not. Happy to do it, but what's the convention/rule-of-thumb?